client/src/components/SubstraitGraph.vue

```diff
@@ -3,6 +3,16 @@
     <div style="margin-top: 10px; list-style: none" id="index" class="fa"></div>
     <div id="svgContainer">
       <div style="margin-right: 10vh; text-align: right">
+        <button
+          ref="download_json"
```

Comment:
Why are there references to `download` in this button?

Reply:
download is a bool value which only becomes `True` if there is a valid d3 plot in the window. This is to avoid having buttons for `Copy Link` and `Download as SVG/PNG` when the d3 plot is not yet generated.